### DISCUSSION TITLE
Add field to indicate silica-preserved tissue sample taken?

### DISCUSSION THREAD

**Discussion by jkleinkopf** [Symbiota/community-board#314] [Wish List]
Hi, I am currently processing a number of collections I made recently, and I noticed there isn't a field available to indicate that I have taken tissue samples of these specimens. I have seen that there is a "Linked Resources" section and an area for linking genetic resources in GenBank, but it would be awesome to have a field (or checkbox) to indicate that tissue samples are available for genetic sequencing.

I'm currently attempting to make coin-envelope-sized specimen labels for tissue samples, but it appears the only way to do this would be via a mail-merge in excel. Right now, I am downloading specimen metadata for each specimen with a tissue sample individually before starting the mail merge. It would be much more efficient to be able to filter by all specimens by tissue availability first! 

Thanks,
Joseph

**Comment by themerekat** [comment]
Thanks for your thoughts, @jkleinkopf ! I wonder if this is what you're looking for: if you do a search through the public interface, you can narrow your search to specimens that have genetic data in the Linked Resources tab
![image](https://user-images.githubusercontent.com/16688784/219210079-991262a9-ff58-4e7a-a8a6-c02136eb54c2.png)

**Reply by jkleinkopf** [reply]
Hi @themerekat, thanks for the quick reply! I looked into using that, but that seems to only be relevant for specimens that have genetic sequence data linked in GenBank. I am trying to find a way to enter data that shows we have taken a silica-preserved tissue sample of a specimen. I have noticed that this is pretty typical in animal divisions (e.g., a liver tissue taken from a mammal specimen) and those data are usually search-able on databases like Arctos. We are building up our tissue collection and it would be ideal to find a way to link specimen-derived tissue samples to the specimen data itself. Thanks!

**Reply by themerekat** [reply]
@jkleinkopf , I don't think there has to be a GenBank link, necessarily, just something present in the "Genetic Resources" box in the Linked Resources tab.
![image](https://user-images.githubusercontent.com/16688784/219463414-d34c458e-83fd-4447-bad1-002037da1cac.png)

**Comment by kyule** [comment]
Hi @jkleinkopf and @themerekat FYI - The NEON Biorepository data portal deals with this in a couple of different ways. One is to have a separate collection for the tissue or DNA samples and the vouchers and use the associatedOccurrences field to link them.  Eg. sometimes there are associatedOccurrences in https://biorepo.neonscience.org/portal/collections/misc/collprofiles.php?collid=28 and https://biorepo.neonscience.org/portal/collections/misc/collprofiles.php?collid=25. This is not exaclty the same as what you are asking but it does allow users to get the same info and means that all tissue samples are searchable. Another thing we do is to use the material samples module / extension to create records for tissues, organs, etc. associated with vouchers that we prepare. This provides users with an indexed list of the available tissue samples associated with the voucher. Material samples are not yet searchable but that is something that is already on the radar for future development.

**Reply by jkleinkopf** [reply]
Hi @kyule, thanks for this! I had not considered using a separate collection for tissue samples and then linking them to specimens data in our portal using the associatedOccurrence field. That makes a lot of sense. It would be great to be able to just click a checkbox -- something like "Limit to Specimens with Preserved Tissue" in the portal -- but in the meantime, we may try your suggestion to make progress on some of our current work.

**Comment by fbungartz** [comment]
I think how this is currently handled is less than adequate and not in line with the extended specimen concept. While all specimens obviously might generate sequence data to link to, subsamples may be different from group to group. For example, chemical secondary metabolite data are very important to lichen specimens, but cannot be linked. For other organims linking that kind of data is not or less relevant. 
A one-size-fits-all makes no sense.
